src/components/Sidebar.tsx

```diff
@@ -49,6 +50,7 @@ import { AttachmentIcon } from '@chakra-ui/icons';
 import { PiFlagCheckeredFill } from 'react-icons/pi';
 
 export const sidebarContentMarginTop = 10;
+export const manageMenuPaddingLeft = 10;
```

Comment:
No need for export

Reply:
`managerDropdownMenuPaddingLeft`

Reply:
And why do you need padding? also if this menu needs padding, the User menu should be padded in the same way sharing variables